### PR TITLE
GeoIP Legacy is EOL: update dev-site redirects

### PR DESCRIPTION
**Pull request by christophermluna**
- `dev.maxmind.com/geoip/legacy` --> `dev.maxmind.com/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired`
- `dev.maxmind.com/geoip/legacy/databases` --> `dev.maxmind.com/geoip/release-notes/2022?lang=en#geoip-legacy-databases-have-been-retired`
- `dev.maxmind.com/geoip/legacy/codes` --> `dev.maxmind.com/geoip/release-notes/2022?lang=en#geoip-legacy-databases-have-been-retired`
- `dev.maxmind.com/geoip/legacy/web-services` --> `dev.maxmind.com/geoip/web-services/legacy`

### CODE REVIEW

File: firebase.js [hunk 1]
```diff
@@ -214,10 +219,15 @@ const config = {
         type: 302,
       },
       {
-        destination: '/geoip/legacy/web-services',
+        destination: '/geoip/web-services/legacy',
         source: '/product/geoip/geoip-legacy-web-services{,/}',
         type: 302,
       },
+      {
```
Comment (dhogan8) [inline firebase.js:226]: Does the `/geoip/web-services/legacy` page exist?

Comment (dhogan8) [inline firebase.js:226]: Does the `/geoip/web-services/legacy` page exist?

File: firebase.js [hunk 2]
```diff
@@ -284,35 +294,45 @@ const config = {
         type: 302,
       },
       {
-        destination: '/geoip/legacy/codes',
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
         source: '/geoip/legacy/codes/country_continent{,/}',
```
Comment (dhogan8) [inline firebase.js:298]: We can probably condense this so that anything starting with `/geoip/legacy/**` gets redirected

Comment (dhogan8) [inline firebase.js:298]: We can probably condense this so that anything starting with `/geoip/legacy/**` gets redirected

File: firebase.js [hunk 3]
```diff
@@ -194,10 +194,15 @@ const config = {
...
       },
       {
-        destination: '/geoip/legacy',
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
         source: '/product/geoip/geoip-legacy{,/}',
         type: 302,
       },
+      {
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
+        source: '/geoip/legacy',
```
Comment (dhogan8) [inline firebase.js:203]: Changing the source to `'/geoip/legacy{,/}'` should allow you to delete all the other redirects that were added in this commit.

File: firebase.js [hunk 4]
```diff
@@ -214,10 +219,15 @@ const config = {
         type: 302,
       },
       {
-        destination: '/geoip/legacy/web-services',
+        destination: '/geoip/web-services/legacy',
         source: '/product/geoip/geoip-legacy-web-services{,/}',
         type: 302,
       },
+      {
+        destination: '/geoip/web-services/legacy',
```
[the remark this answers is not in the document]
Comment (christophermluna) [reply firebase.js:227]: drat, i see that it should be `/geoip/docs/web-services/legacy

**Review by dhogan8 — APPROVED** [review]
Looks good!

**Pull request merged**